stage1/gc/gc.go

```diff
@@ -33,8 +36,13 @@ import (
 	rktlog "github.com/coreos/rkt/pkg/log"
 )
 
+const (
+	cgroupFsPath = "/sys/fs/cgroup"
+)
+
 var (
 	debug bool
+	log   *rktlog.Logger = rktlog.New(os.Stderr, "stage1 gc", debug)
```

Comment:
This is usually assigned in one file and other files in the same package still use `log`, so go-imports then imports the stdlib `log` pkg (it doesn't know that `log` is defined elsewhere in the same pkg)